### ISSUE TITLE
Bug if we use it in a sonata admin tab

### ISSUE THREAD

**Issue by SebBareyre** [ollietb/OhGoogleMapFormTypeBundle#22]
![ohgoogle](https://cloud.githubusercontent.com/assets/3274119/4997552/4d554ca0-69d0-11e4-93ad-8e86c1b38dfa.jpg)

Hello, 
I have a bug when i use this bundle in a sonata admin tab. 
Could you help me to solve this problem.

Thanks
Seb

**Comment by SebBareyre** [comment]
I have set width and height and it doesn't work either. I think the problem come from the map is located in a hidden tab at page loading... no ?

**Comment by ollietb** [comment]
@SebBareyre yes, this will happen if you try to load the map before it is visible

**Comment by SebBareyre** [comment]
ok and how we could solve that ?

**Comment by SebBareyre** [comment]
up

**Comment by jan-j** [comment]
@SebBareyre After showing tab with map you need to trigger following code:

```js
var center = map.getCenter(); 
google.maps.event.trigger(map, 'resize'); 
map.setCenter(center);
```

 This will refresh map and everything should be fine then. Middle line is significant here, other two are only to ensure center of the map will stay the same.

**Comment by xmon** [comment]
Any idea to trigger this code on click tab in sonataadmin?
